poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
_Suggestion_: Since we are not adding any additional library anymore, can you check if this change on the poetry.lock is required?